src/platform-includes/performance/opentelemetry-setup/node.mdx

```diff
@@ -17,25 +25,81 @@ const {
+const {
+  AsyncLocalStorageContextManager,
+} = require("@opentelemetry/context-async-hooks");
 
-// Make sure to call `Sentry.init` BEFORE initializing the OpenTelemetry SDK
-Sentry.init({
-  dsn: "___PUBLIC_DSN___",
-  tracesSampleRate: 1.0,
-  // set the instrumenter to use OpenTelemetry instead of Sentry
-  instrumenter: "otel",
-  // ...
-});
+function setupSentry() {
+  setupGlobalHub();
+
+  // Make sure to call `Sentry.init` BEFORE initializing the OpenTelemetry SDK
+  Sentry.init({
+    dsn: "___PUBLIC_DSN___",
+    tracesSampleRate: 1.0,
+    // set the instrumenter to use OpenTelemetry instead of Sentry
+    instrumenter: "otel",
+    // ...
+  });
+
+  const client = getCurrentHub().getClient();
+  setupEventContextTrace(client);
+
+  // You can wrap whatever local storage context manager you want to use
+  const SentryContextManager = wrapContextManagerClass(
+    AsyncLocalStorageContextManager
+  );
+
+  const sdk = new opentelemetry.NodeSDK({
+    // Existing config
+    traceExporter: new OTLPTraceExporter(),
+    instrumentations: [getNodeAutoInstrumentations()],
 
-const sdk = new opentelemetry.NodeSDK({
-  // Existing config
-  traceExporter: new OTLPTraceExporter(),
-  instrumentations: [getNodeAutoInstrumentations()],
+    // Sentry config
+    spanProcessor: new SentrySpanProcessor(),
+    textMapPropagator: new SentryPropagator(),
+    contextManager: new SentryContextManager(),
+    sampler: new SentrySampler(client),
+  });
 
-  // Sentry config
-  spanProcessor: new SentrySpanProcessor(),
-  textMapPropagator: new SentryPropagator(),
+  // Ensure OpenTelemetry Context & Sentry Hub/Scope is synced
+  setOpenTelemetryContextAsyncContextStrategy();
+
+  sdk.start();
+}
+
+setupSentry();
+```
+
+Note that with this setup, you can _only_ use OpenTelemetry tracing for performance. `Sentry.startTransaction()` will _not_ work.
+You can either create spans via `tracer.startActiveSpan()`,
+or use the `startSpan()` / `startInactiveSpan()` methods exported from `@sentry/opentelemetry`,
+which are just thin wrappers around the OpenTelemetry API:
+
+```js
+import { startSpan, startInactiveSpan } from "@sentry/opentelemetry";
+
+startSpan({ name: "my span" }, (span) => {
+  // span is an OpenTelemetry Span!
+  span.setAttribute("my_attr", "value");
+  // span is automatically ended at the end of the callback
 });
 
-sdk.start();
+const span = startInactiveSpan({ name: "my span" });
+// do something
+span.end();
 ```
+
+<Note>
+  If you are using the{" "}
+  <a
+    href="https://www.npmjs.com/package/@sentry/opentelemetry-node"
+    target="_blank"
+    rel="noopener"
+  >
+    @sentry/opentelemetry-node
+  </a>{" "}
+  package, you can continue to do so and find usage instructions in the
+  package's README. This package has a slightly easier setup process, but
+  provides a more limited connection between Sentry and OpenTelemetry than the
+  instructions outlined above.
+</Note>
```

Comment:
I think if you leave extra newlines padding the contents of the note, markdown inside the note will render correctly:

```suggestion
<Note>

  If you are using the [@sentry/opentelemetry-node](https://www.npmjs.com/package/@sentry/opentelemetry-node] package, you can continue to do so. Usage instructions are in the
  package's README. This package has a slightly easier setup process but
  provides a more limited connection between Sentry and OpenTelemetry than the
  instructions outlined above.
  
</Note>
```

Reply:
ah, nice, didn't know that! 👍